SeriesMaster-Frontend/App/Screens/SettingsScreen.js

```diff
@@ -0,0 +1,22 @@
+import Screen from "./Screen";
+import React, { useContext } from "react";
+import AppButton from "../components/AppButton";
+import tokenStorage from "../auth/tokenStorage";
```

Comment:
destructure this import for better readability

Reply:
Thanks!